### DISCUSSION TITLE
在服务器上安装alsit，然后挂载夸克TV后，使用302方式，还是走的服务器端的流量和速度

### DISCUSSION THREAD

**Discussion by xiaoyao20084321** [AlistGo/alist#7227] [General]
你好，大家有没有遇到过，在服务器上安装alsit，然后挂载夸克TV后，使用302方式，还是走的服务器端的流量和速度

**Comment by anwen-anyi** [comment]
检查你是否开启了`Web代理`或者`Webdav策略的本地代理`

**Reply by xiaoyao20084321** [reply]
<img width="1296" alt="image" src="https://github.com/user-attachments/assets/b6da78fd-02e2-46cb-a8b5-f18e0a24f878">
确定没开启。

**Comment by mark-liu0000** [comment]
115网盘也是这样最新版v3.37.4，也是走的本地带宽

**Reply by anwen-anyi** [reply]
很明显你开启了代理导致的...
 - 检查你是否开启了`Web代理`或者`Webdav策略的本地代理`

**Reply by mark-liu0000** [reply]
意思是关闭web代理，但是之前也不走本地带宽啊，是因为115换了接口吗？

**Comment by Bennie-CHEN** [comment]
楼主解决了吗？我115正常能302，可是夸克TV 默认302可是走不了直链

**Reply by xiaoyao20084321** [reply]
没解决。。夸克TV302也走不了直连。后来懒得弄了。。

**Comment by LINJIAXIN-China** [comment]
我现在测试是正常的302的，alist版本是3.38

**Reply by Bennie-CHEN** [reply]
我也是3.38，夸克TV确实没有直连，我用ddnsto所以很明显能看出来走了宿主机转发

**Reply by CODPHISH** [reply]
> 我也是3.38，夸克TV确实没有直连，我用ddnsto所以很明显能看出来走了宿主机转发

我是3.39.1，发现302被限速着，但用tvbox的看别人分享的4K就很流畅

**Reply by 1103020472** [reply]
我的也是，直接在电脑上通过raidrive看4k就很流畅，但是通过其他第三方webdav软件就被限速

**Comment by Rafeal-Ma** [comment]
楼上解决了吗？为什么我的302速度只有几百KB

**Reply by verydemo** [reply]
我也没办法走302，

**Comment by qcnhy** [comment]
不论是302还是 代理都被限速128k
